src/QueryBuilder/DSL/Query.php

```diff
@@ -72,9 +72,9 @@ public function getType(): string
      *
      * @param mixed $values
      */
-    public function match(?string $field = null, $values = null): Match
+    public function match(?string $field = null, $values = null): Match_
```

Comment:
BC break here, but impact should be very low.

Reply:
Not for users running php 7.4 and up.

As for them if they extend it they can speciyf their method returns the `Match` class.

Reply:
It won't impact users on 7.4 just for covariance.
But probability that users care about this return is very low ;)
By care I mean `instanceof` or anything that is tied to the previous class name